docs/openapi/components/schemas/common/PGARelease.yml

```diff
@@ -0,0 +1,27 @@
+$linkedData:
+  term: PGARelease
+  '@id': https://w3id.org/traceability#PGARelease
+title: PGARelease
+description: Status document to release cargo
+type: object
+required:
+  - type
+  - reasonCode
+properties:
+  type:
+    oneOf:
+      - type: array
+      - type: string
+        enum:
+          - PGARelease
+  reasonCode:
+    type: string
+  description:
+    type: string
+additionalProperties: true
+example: |-
+  {
+    "type": "PGARelease",
+    "reasonCode" : "07",
```

Comment:
@BenjaminMoe , are you sure that a code is needed/provided for a release?

Reply:
Good point. I'll remove the reason and description on the release and update the PR from draft to review.